app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(16, 4)) == 16
```

Comment:
The division in the assertion should use integer division to ensure the sum of parts is an integer. Use `16 // 4` instead of `16 / 4`.